### DISCUSSION TITLE
Feature Request: Statistics Card - show data only within a single day

### DISCUSSION THREAD

**Discussion by Jimbo222** [home-assistant/frontend#18744] [Dashboard]
Can I request to add the option on the Statistics Graph card to show only data within a single day. i.e. starting from 00:00 until current time.  Presently there is only the option to show the last X.X days.

Thanks

**Comment by ildar170975** [comment]
I wonder why do you need Statistics graph card for such a short period ("today") instead of History graph card?
Is it only because of showing "min" & "max" curves?

**Reply by Jimbo222** [reply]
I like to options for lower sampling rates.  A few of my graphs have values that jump around a bit so the smoother line makes it a little more readable.  Min and Max are also useful sometimes.

Does the history graph have this feature? I didn't see it.
